packages/reason-standard/reason-standard.0.1.0-alpha.1/opam

```diff
@@ -0,0 +1,38 @@
+opam-version: "2.0"
+synopsis: "A portable standard library enhancement for Reason and OCaml."
+description: "A portable standard library enhancement for Reason and OCaml."
+maintainer: ["deanmerchant@gmail.com"]
+authors: ["Dean Merchant"]
+license: "MIT"
+homepage: "https://github.com/dean177/reason-standard"
+bug-reports: "https://github.com/dean177/reason-standard/issues"
+depends: [
+  "alcotest" {with-test}
+  "base" {>= "0.13.0"}
+  "zarith" {>= "1.9.1"}
+  "dune" {>= "2.4"}
+  "reason" {>= "3.4.0"}
+]
```

Comment:
```suggestion
  “ocaml” {>= "4.08.0"}
]
```

Reply:
After all there cannot be an opam switch without an ocaml compiler, so it is not adding extra dependencies

Reply:
Thanks for your help @mseri